README.md

```diff
@@ -1,22 +1,22 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# TNW-Responsive-Clone
 
-> One paragraph statement about the project.
+> Creating a responsive clone of TNW (thenextweb.com) website. Projects includes differnet stylings of screens i.e. Moble UI, Tablet UI, Desktop UI. Dynamic images and content.
 
 ![screenshot](./app_screenshot.png)
```

Comment:
- [ ] Fix your screenshot, it's not showing in your README file.

Reply:
Fixed!